### PR TITLE
Part4

### PR DESCRIPTION
**Pull request by albanD**

### CODE REVIEW

**Comment by albanD** [comment]
@bunelr should be all done.

File: test/main.cpp [hunk 1]
```diff
@@ -333,12 +338,57 @@ int check_end_part_3() {
...
+    myGrid.getCell(0,1) = 2;
+    cout<<"Testing if grid is consistent, should be 0, result: "<<myGrid.isConsistent()<<endl;
+    errors += (myGrid.isConsistent()!=0);
+
+    content_col= {
+        Region("6---359--"), Region("72-169-8-"), Region("-318-----"),
+        Region("-56--134-"), Region("-9----5--"), Region("----7-2--"),
+        Region("--8-7----"), Region("6-3-5----"), Region("75---3-8-")
+    };
+    content_col= {
```
Comment (bunelr) [inline test/main.cpp:373]: why is this one directly overwriting the other?

Comment (albanD) [inline test/main.cpp:373]: because the other one is too weak/easy, survival of the fittest

**Comment by bunelr** [comment]
Awesome job! 
image

**Pull request merged**